test/integration/ProjectBrowserTest.kt

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+// We need to access private function Studio.MainWindowColumn, this allows us to.
+// Do not use this outside of tests anywhere. It is extremely dangerous to do so.
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
+
+package com.vaticle.typedb.studio.test.integration
+
+import java.util.UUID
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.vaticle.typedb.studio.state.StudioState
+import org.junit.Rule
+import org.junit.Test
+
+class ProjectBrowserTest {
+    @get:Rule
+    val composeRule = createComposeRule()
+
+    @Test
+    fun createADirectory() {
+        val uuid = UUID.randomUUID().toString()
+        studioTest(composeRule) {
+            val createdDirectoryName = "created"
+
+            cloneAndOpenProject(composeRule, SAMPLE_DATA_PATH, uuid)
```

Comment:
Let's leverage Kotlin's named parameters to clarify the meaning of this call.
```suggestion
            cloneAndOpenProject(composeRule = composeRule, path = SAMPLE_DATA_PATH, name = uuid)
```
Or, if we also adopt the renaming of `uuid` to `testID`:
```suggestion
            cloneAndOpenProject(composeRule = composeRule, path = SAMPLE_DATA_PATH, name = testID)
```
By naming these parameters, a reader will infer:
1. To understand `composeRule` will need contextual knowledge gained by studying the code more;
2. `SAMPLE_DATA_PATH` is likely to be the project path;
3. `uuid`/`testID` is likely to be the project name.

If any of these assumptions are incorrect, that suggests that the parameters of `cloneAndOpenProject` should be renamed; if not, then its signature is a strong signature.

Reply:
We've reworked some of the signatures to be more clear. We've standardised on `dbName` for the name of the database. As for `cloneAndOpenProject` we've renamed the second and third arguments to `source` and `destination` as it wasn't perfectly clear what it did, given your comment.